compiler/ast/ast.go

```diff
@@ -377,6 +377,14 @@ type (
 		Kind   string  `json:"kind" unpack:""`
 		Trunks []Trunk `json:"trunks"`
 	}
+	Load struct {
+		Kind    string `json:"kind" unpack:""`
+		Pool    string `json:"pool"`
+		Author  string `json:"author"`
+		Message string `json:"message"`
+		Meta    string `json:"meta"`
+		Branch  string `json:"branch"`
```

Comment:
Put Branch after Pool since they're coupled.
```suggestion
		Pool    string `json:"pool"`
		Branch  string `json:"branch"`
		Author  string `json:"author"`
		Message string `json:"message"`
		Meta    string `json:"meta"`
```